packages/client/src/components/GrantsTable.vue

```diff
@@ -6,7 +6,7 @@
           <b-input-group-text>
             <b-icon icon="search" />
           </b-input-group-text>
-          <b-form-input type="sliders" @input="debounceSearchInput"></b-form-input>
+          <b-form-input type="text" @input="debounceSearchInput"></b-form-input>
```

Comment:
Nit from previous code - this caused prop type warnings.

Reply:
Looks good 👍